charts/kubewarden-defaults/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: kubewarden-policy-server
```

Comment:
I'm confused, the chart is called `kubewarden-policy-server`, but lives inside of the `kubewarden-defaults` directory.

Reply:
Sorry. I forgot to update the name in the `Chart.yaml` after renaming the directory.